docs/reference/commandline/run.md

```diff
@@ -479,3 +479,16 @@ available in the default container, you can set these using the `--ulimit` flag.
 
 The values are sent to the appropriate `syscall` as they are set.
 Docker doesn't perform any byte conversion. Take this into account when setting the values.
+
+#### For `nproc` usage:
+
+Be careful setting `nproc` with the `ulimit` flag as `nproc` is designed by Linux to set the maximum number of processes available to a user, not to a container.  For example, start four containers with `daemon` user:
+
+
+    docker run -d -u daemon --ulimit nproc=3 busybox top
+    docker run -d -u daemon --ulimit nproc=3 busybox top
+    docker run -d -u daemon --ulimit nproc=3 busybox top
+    docker run -d -u daemon --ulimit nproc=3 busybox top // fail
```

Comment:
I don't think `//` is a valid way to add a comment. Either change it to `# fail`, or simply remove `// fail`.

Removing it should be fine, because we explain what happens in the paragraph below